ops/ansible/roles/bfd-db-migrator/defaults/main.yml

```diff
@@ -2,7 +2,6 @@
 env: test
 db_migrator_dir: /opt/bfd-db-migrator
 db_migrator_user: bb-migrator
-db_migrator_jvm_args: -Xmx64g
```

Comment:
curious as to why you removed the mem config.

Reply:
tl;dr; It's unused.

The values and shape of the `bfd-db-migrator` role were initially copied from `bfd-pipeline` because the migrator is going to be taking over for some of the functionality that the pipeline is currently doing, so I reasoned that the solution for its role would be similar in many ways. As it turns out, the pipeline doesn't actually make use of that value, and because pipeline was effectively the template, it's not used in the migrator, either.